src/desktop/apps/notifications/templates/mobile/index.jade

```diff
@@ -1,5 +1,5 @@
-#notifications-page.main-side-margin
-  h1.notifications-header.avant-garde-header-center
+#notifications-page.mobile
```

Comment:
So that I could target just the mobile view, since both the mobile and desktop stylesheets are included in both views.